data/profiles/defaultboot.ipxe

```diff
@@ -0,0 +1,4 @@
+kernel <%=url%>/<%=kernel%>
```

Comment:
@anhou , could you please explain how the defualtboot.ipxe will be used? I see you created this new ipxe script, but nowhere will use it in your code.